resources/hu/cemu.po

```diff
@@ -57,6 +57,122 @@ msgstr "Írj be egy memória címet"
 msgid "Memory breakpoint"
 msgstr "Memória szünet"
 
+#: CemuApp.cpp
+msgid "Browse"
+msgstr "Tallózás"
+
+#: CemuApp.cpp
+msgid "Select a file"
+msgstr "Válassz ki egy fájlt"
+
+#: CemuApp.cpp
+msgid "Select a directory"
+msgstr "Válassz ki egy könyvtárat"
+
+#: CemuApp.cpp
+msgid "base"
+msgstr "alap"
+
+#: CemuApp.cpp
+msgid "update"
+msgstr "frissítés"
+
+#: CemuApp.cpp
+msgid "dlc"
+msgstr "dlc"
+
+#: CemuApp.cpp
+msgid "save"
+msgstr "mentés"
+
+#: CemuApp.cpp
+msgid "Japan"
+msgstr "Japán"
+
+#: CemuApp.cpp
+msgid "USA"
+msgstr "Egyesült Államok"
+
+#: CemuApp.cpp
+msgid "Europe"
+msgstr "Európa"
+
+#: CemuApp.cpp
+msgid "Australia"
+msgstr "Ausztrália"
+
+#: CemuApp.cpp
+msgid "China"
+msgstr "Kína"
+
+#: CemuApp.cpp
+msgid "Korea"
+msgstr "Korea"
+
+#: CemuApp.cpp
+msgid "Taiwan"
+msgstr "Tajvan"
+
+#: CemuApp.cpp
+msgid "Auto"
+msgstr "Automatikus"
+
+#: CemuApp.cpp
+msgid "many"
+msgstr "sok"
```

Comment:
This "many" refers to multi-region titles (since they have "many" regions), so there is probably a better translation for it.